ocaml/xcp-rrdd/bin/rrdd/rrdd_server.ml

```diff
@@ -162,6 +162,10 @@ let backup_rrds (remote_address : string option) () : unit =
     )
   done
 
+let save_rrds =
+  info "%s: trying to save RRDs to localhost" __FUNCTION__ ;
+  backup_rrds None
```

Comment:
Do we actually need a new function. It looks like `backup_rrds` is there and already sufficient?

Reply:
backup_rrds normally takes in a remote (the pool master), In this case we are passing in None to trigger a local save

Reply:
Yes, `backup_rrds` in the idl can also take `None`. The CLI can call it with `None`.